bitcoinkit/src/main/kotlin/io/horizontalsystems/bitcoinkit/network/MainNet.kt

```diff
@@ -43,4 +43,6 @@ class MainNet : NetworkParameters() {
     override fun validateBlock(block: Block, previousBlock: Block) {
         blockValidator.validate(block, previousBlock)
     }
+
+    override val explorerUrl: String = "https://chain.api.btc.com/v3"
```

Comment:
Let's store explorer urls in ApiRequesterBtcCom and choose url depending on network id